### PR TITLE
Add due label to the name of the documents

### PR DESCRIPTION
**Pull request by Adnanog**

### CODE REVIEW

File: client/src/components/Dashboard.js [hunk 1]
```diff
@@ -107,9 +108,29 @@ class Dashboard extends Component {
...
+                const dateOnly = moment(currentDate).format("L");
+                const threeDaysLeft = moment(currentDate)
+                  .add(3, "days")
+                  .format("L");
+                const twoDaysLeft = moment(currentDate)
+                  .add(2, "days")
+                  .format("L");
+                const oneDayLeft = moment(currentDate)
+                  .add(1, "days")
+                  .format("L");
```
Comment (llh1) [inline client/src/components/Dashboard.js]: All of these variable don't need to be defined here as they are only used inside NameColumn component. You can just pass "currentDate" as a props to NameColumn component and inside the component define all these variables.

File: client/src/components/NameColumn.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return (
+      <Table.Cell>
+        {name} <Label color="red">Due in 3 days</Label>
+      </Table.Cell>
+    );
+  } else if (status === "Waiting" && dueDate === dateOnly) {
+    return (
+      <Table.Cell>
+        {name} <Label color="red">Due today</Label>
+      </Table.Cell>
```
Comment (llh1) [inline client/src/components/NameColumn.js]: All this code above have duplication, how about you keep you just if else if for the value inside the label?

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/Dashboard.js (hunk 1 above)
Comment (Adnanog) [reply client/src/components/Dashboard.js]: ok

File: client/src/components/NameColumn.js (hunk 2 above)
Comment (Adnanog) [reply client/src/components/NameColumn.js]: it is done

**Pull request merged**